memote/suite/results/result_manager.py

```diff
@@ -76,13 +76,16 @@ def store(self, result, filename, env_info=True, pretty=True):
         if env_info:
             self.add_environment(result.meta)
         if pretty:
-            kwargs = dict(sort_keys=True, indent=2, separators=(",", ": "))
+            kwargs = dict(sort_keys=True, indent=2,
+                          separators=(",", ": "), ensure_ascii=True)
         else:
-            kwargs = dict(sort_keys=False, indent=None, separators=(",", ":"))
+            kwargs = dict(sort_keys=False, indent=None,
+                          separators=(",", ":"), ensure_ascii=True)
         LOGGER.info("Storing result in '%s'.", filename)
         with open(filename, "w", encoding="utf-8") as file_handle:
             try:
-                return json.dump(result, file_handle, **kwargs)
+                return file_handle.write(
+                    str(json.dumps(result, **kwargs)))
```

Comment:
If you set `ensure_ascii` to `False` above, `json.dumps` returns unicode automatically and you don't need to convert it in this line.